### PR TITLE
ansible: refresh macOS test machines + playbook patches

### PR DESCRIPTION
**Pull request by gdams**
##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md#commit-messages)
- [ ] FAQ.md updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [x] inventory changes, ensure bastillion is updated accordingly

### CODE REVIEW

**Comment by karianna** [comment]
Linter fixes needed.

**Comment by karianna** [comment]
```
TASK [Common : Upgrade installed packages] *************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Updating Homebrew...\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nError: Could not remove ansible keg! Do so manually:\n  sudo rm -rf /usr/local/Cellar/ansible/2.9.13_1"}
```

**Comment by sxa** [comment]
Is there any reason these cannot be added to Bastillion (as per the last checkbox)?

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Jenkins_User/tasks/main.yml [hunk 1]
```diff
@@ -5,8 +5,19 @@
...
+- name: Create Jenkins user (macOS)
+  action: user name={{ Jenkins_Username }} group=staff state=present shell=/bin/bash
+  when: ansible_distribution == "MacOSX"
+  tags: [jenkins_user, adoptopenjdk]
+
+- name: Enable SSH login for Jenkins user (macOS)
+  command: dseditgroup -o edit -a {{ Jenkins_Username }} -t staff com.apple.access_ssh
+  ignore_errors: yes
+  when: ansible_distribution == "MacOSX"
+
```
Comment (Haroon-Khel) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Jenkins_User/tasks/main.yml:20]: `jenkins` and `adoptopenjdk` tags on this task would be appropriate

**Review by Haroon-Khel — CHANGES_REQUESTED** [review]

**Comment by Haroon-Khel** [comment]
The changes made to the inventory file would put it in sync with the machines listed in Jenkins, as per https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1675, so Im all for that. 

However, I do not have a fresh mac os machine on which to test out the ansible changes

**Comment by gdams** [comment]
@Haroon-Khel updated PTAL

**Comment by karianna** [comment]
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Updating Homebrew...\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nNo entry for terminal type \"unknown\";\nusing dumb terminal settings.\nError: Could not remove ansible keg! Do so manually:\n  sudo rm -rf /usr/local/Cellar/ansible/2.9.15"}

**Comment by gdams** [comment]
> Is there any reason these cannot be added to Bastillion (as per the last checkbox)?

@sxa I've now added all macs from the inventory file into Bastillion 🎉

**Comment by karianna** [comment]
run tests

**Review by Haroon-Khel — APPROVED** [review]
The ansible additions look good to me, they pass on the macos check. The new Xcode10 role doesnt run on the macos check, since it skips roles tagged with `adoptopenjdk`, but @gdams confirms that he has tested it and it works. Same goes for the macos_codesign role.

**Pull request merged**